openIMIS/openIMIS/settings.py

```diff
@@ -347,3 +347,13 @@ def SITE_ROOT():
         },
     },
 }
+
+# Django email settings
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+
+EMAIL_HOST = os.environ.get('EMAIL_HOST', 'localhost')
+EMAIL_PORT = os.environ.get('EMAIL_PORT', '1025')
```

Comment:
1025 ? SMTP is  25, 587, 465 or 2525

Reply:
Yes, you are absolutely right! I'm about to change this default port.

Reply:
@delcroip I simply used debugging server as a default during developing so as to check if email is sent "python -u -m smtpd -n -c DebuggingServer localhost:1025" according to the "Configuring email for development" section on the bottom of this webpage https://docs.djangoproject.com/en/3.1/topics/email/).